src/lumigo_tracer/utils.py

```diff
@@ -369,62 +368,89 @@ def md5hash(d: dict) -> str:
     return h.hexdigest()
 
 
+class IntermediateOmitResult:
```

Comment:
It can be private, isn't it?
```suggestion
class _IntermediateOmitResult:
```